### PR TITLE
File routing table

### PR DESCRIPTION
**Pull request by SzymonPobiega**
Adds doco on file-based routing and adds a missing section on static routes to the main routing doco.

### CODE REVIEW

File: nservicebus/messaging/file-based-routing.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+title: File-based message routing
+summary: How to configure file-based message routing in NServiceBus
+tags:
+- routing
+- message
+- route
+- file
```
Comment (SimonCropp) [inline nservicebus/messaging/file-based-routing.md]: The tag xml file and dynamic dont make sense here. Message is too generic

File: nservicebus/messaging/file-based-routing.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+- nservicebus/messaging/routing
+---
+
+Before version 6, NServiceBus used fixed [message ownership mappings](/nservicebus/messaging/message-owner.md) in the standard .NET configuration file. The advantage was that operations personnel can adjust the routing should the topology change. This is especially important in case of bus-like transports (e.g. MSMQ) where machine name is part of the routing information. Version 6 introduced all-new [code-first API](/nservicebus/messaging/routing.md) that allows for better developer experience. In order to provide a better experience also for operations, we introduced file-based routing.
+
+## Configuration
+
+The following code
+
+snippet:Routing-FileBased-Config
```
Comment (SimonCropp) [inline nservicebus/messaging/file-based-routing.md]: Placing a snippet inside a sentence will make for some fairly disjointed reading

**Comment by SzymonPobiega** [comment]
@SimonCropp removed the tags and reworded that section.

**Comment by SimonCropp** [comment]
@SzymonPobiega i notice u added a snippet to an xsd file. currently that is not allowed in the file types scanned for snippets. i will update the docs engine to add it and get back to u

**Pull request merged**